# Results Summary – Volatility Crush Study

## Objective
This notebook consolidates results and presents
the final economic and trading insights.

In [7]:
import sys
from pathlib import Path
import pandas as pd

PROJECT_ROOT = Path.cwd().parent
sys.path.insert(0, str(PROJECT_ROOT))

from src.analysis.metrics import performance_summary, ticker_summary

In [8]:
df = pd.read_csv(PROJECT_ROOT / "data" / "earnings_simulated.csv")
df.head()

Ticker  Straddle_PnL  IronCondor_PnL
0   AAPL      8.615937       -4.490565
1   AAPL     -0.160302        0.003946
2   AAPL      0.895562       -0.051751
3   AAPL     10.472456       -5.303927
4   AAPL      2.159368       -0.131674

## Overall Strategy Performance

The table below summarizes aggregate performance across all earnings events.

In [9]:
summary_df = pd.DataFrame(
    [
        performance_summary(df["Straddle_PnL"]),
        performance_summary(df["IronCondor_PnL"])
    ],
    index=["Straddle", "Iron Condor"]
)

summary_df

Total Trades  Mean PnL  Median PnL  Win Rate    Total PnL  \
Straddle              400  4.798326    3.780126    0.8150  1919.330343   
Iron Condor           400 -1.973681   -1.094287    0.3025  -789.472348   

             Max Drawdown  
Straddle         3.185382  
Iron Condor    784.985729

## Interpretation: Aggregate Performance

### Straddle (Long Volatility)
- **Total Trades:** 400
- **Win Rate:** 81.5%
- **Mean PnL:** +4.80
- **Total PnL:** +1919
- **Max Drawdown:** 3.18

Despite a high win rate, the straddle strategy relies on
**frequent small gains** and is highly sensitive to
implied volatility assumptions.

### Iron Condor (Short Volatility)
- **Total Trades:** 400
- **Win Rate:** 30.3%
- **Mean PnL:** −1.97
- **Total PnL:** −789
- **Max Drawdown:** 785

Although theoretically aligned with volatility crush,
the iron condor exhibits **fat-tail losses** and
significant drawdowns under adverse price moves.

## Key Takeaways

1. **Volatility crush alone does not guarantee profitability**
2. Strategy payoff convexity is critical during earnings
3. Long volatility benefits from:
   - Fat-tailed price jumps
   - Asymmetric payoffs
4. Short volatility strategies suffer from:
   - Low win rate
   - Severe tail losses

## Final Conclusion

This study demonstrates that while earnings events are associated
with implied volatility collapse, **strategy design matters more than direction**.

Although iron condors are theoretically aligned with volatility crush,
their concave payoff structure exposes them to severe tail risk.

Conversely, long straddles benefit from convex payoffs
that dominate volatility decay during large earnings surprises.

**Conclusion:**  
Earnings trading is not simply about volatility levels,
but about **distributional asymmetry and payoff convexity**.